app-ios/tutanota/Sources/GeneratedIpc/CommonNativeFacade.swift

```diff
@@ -1,6 +1,5 @@
 /* generated file, don't edit. */
```

Comment:
The linter should not have touched the generated files. Unfortunately, we do not have a way yet to easily run the full linting command (as seen in the XCode build stage) nor is it documented.